deployments/data-hub/release-data-hub--prod.yaml

```diff
@@ -129,9 +129,9 @@ spec:
         value: "0 7 * * 1-5"
       - name: CROSS_REF_IMPORT_SCHEDULE_INTERVAL
         value: "@daily"
-        # At minute 20 past every hour from 7 through 20 on every day-of-week from Monday through Friday.
+        # At minute 20 past every hour from 6 through 19 on every day-of-week from Monday through Friday.
       - name: GOOGLE_SPREADSHEET_SCHEDULE_INTERVAL
-        value: "20 7-20/1 * * 1-5"
+        value: "20 6-19/1 * * 1-5"
```

Comment:
Should we perhaps just move the start time?
(The description mentioned to be able to look at it earlier in the morning)

Reply:
I don't know what the actual intention was of the change. Is it to account for summer time and previous settings where for winter time?